home-assistant-addon/nginx.conf

```diff
@@ -29,11 +32,36 @@ http {
     # 1. Strip the ingress prefix before proxying to Mastra
     # 2. Rewrite responses to add the prefix back for assets/links
     # 3. Handle WebSocket connections
+    # 4. Apply JWT authentication only to MCP server endpoints
     server {
         server_name 0.0.0.0;
         listen 5690;
```

Comment:
This port is not the one used in the Home Assistant addon for the ingress.

I also still would like the Mastra UI and MCP servers to have separate ports in Nginx, which makes it easier to control certain reverse tunnels etc.

I'd like the ports to be the same as today to the outside, so Nginx should use 4111 and 4112, but then the internal ports used inside the Mastra process and the Bun process should be something else.